examples/vitepress/docs/index.md

```diff
@@ -0,0 +1 @@
+# Hello VitePress
```

Comment:
Lets add another page to make sure it works. Its not clear if this is a SPA or SSG (because the former would need routes)